File: 1-js-basics/3.2-conditional-statements-advanced-exercise/07-hotelRoom.js
function solve([month, nights]) {

    nights = Number(nights);

    let apartment = 0;

    let studio = 0;

    if (month == 'May' || month == 'October') {

        if (nights > 7 && nights <= 14) {

            studio = (50 * nights) * 0.95;

            apartment = 65 * nights;

        } else if (nights > 14) {

            studio = (50 * nights) * 0.70;

            apartment = (65 * nights) * 0.90;

        } else {

            studio = 50 * nights;

            apartment = 65 * nights;
        }

    } else if (month == 'June' || month == 'September') {

        if (nights > 14) {

            studio = (75.20 * nights) * 0.80;

            apartment = (68.70 * nights) * 0.90;

        } else {

            studio = 75.20 * nights;

            apartment = 68.70 * nights;

        }

    } else {

        if (nights > 14) {

            apartment = (77 * nights) * 0.90;

        } else {

            apartment = 77 * nights;
            
        }

        studio = 76 * nights;

    }

    return `Apartment: ${apartment.toFixed(2)} lv.\nStudio: ${studio.toFixed(2)} lv.`;

}

console.log(solve(['May', '15']));
console.log(solve(['June', '14']));
console.log(solve(['August', '20']));

File: 2-python/1-python-basics/3.2-conditional-statements-advanced-exercise/07-hotel_room.py
month_name = input()
nights_count = int(input())
apartment_price = nights_count
studio_price = nights_count

if month_name == 'May' or month_name == 'October':
    apartment_price *= 65
    studio_price *= 50
    if 7 < nights_count <= 14:
        studio_price *= 0.95
    elif nights_count > 14:
        studio_price *= 0.7
elif month_name == 'June' or month_name == 'September':
    apartment_price *= 68.7
    studio_price *= 75.2
    if nights_count > 14:
        studio_price *= 0.8
else:
    apartment_price *= 77
    studio_price *= 76

if nights_count > 14:
    apartment_price *= 0.9

print(f'Apartment: {apartment_price:.2f} lv.\nStudio: {studio_price:.2f} lv.')
